browser/ntp_background_images/android/ntp_background_images_bridge.cc

```diff
@@ -200,3 +200,8 @@ void NTPBackgroundImagesBridge::OnUpdated(NTPBackgroundImagesData* data) {
   JNIEnv* env = AttachCurrentThread();
   Java_NTPBackgroundImagesBridge_onUpdated(env, java_object_);
 }
+
+void NTPBackgroundImagesBridge::OnSuperReferralEnded() {
+  JNIEnv* env = AttachCurrentThread();
```

Comment:
@deeppandya Do you think it's sufficient to call `_onUpdated()` only here?

Reply:
@simonhong in android, we don't use observers. all we look for is whether we get an image and meta data or not. so in that context it should be enough.

Reply:
Thanks. Will make method empty here :)